packages/rpc-spec/src/rpc-api.ts

```diff
@@ -3,7 +3,7 @@ import { Callable } from '@solana/rpc-spec-types';
 import { RpcRequest } from './rpc-request';
 
 export type RpcApiConfig = Readonly<{
-    parametersTransformer?: <T extends unknown[]>(params: T, methodName: string) => unknown[];
+    parametersTransformer?: <T extends unknown[]>(params: T, methodName: string) => unknown;
```

Comment:
I think so, because the developer doesn't control that - it's just the input params to their function. From a JS perspective that'll always be a list, I think.